### PR TITLE
Add beta for org.signal.Signal

### PR DESCRIPTION
**Pull request by cam-rod**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
  - _Contacted the developers two weeks ago, have not received a response yet._
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*
  - `signal-desktop-beta.sh` is required to pass the correct options to the Electron app, and launch it within the Flatpak. This is the same as what is done in the stable flatpak.

This is a PR to become the maintainer for the beta of [Signal Desktop](https://github.com/signalapp/Signal-Desktop), in parallel with the stable Flathub release
at [flathub/org.signal.Signal](https://github.com/flathub/org.signal.Signal_Beta). It is a minimally modified version of the existing release (similar to #1798), with
some updated metadata that could also be added to the stable branch. I've automated builds and tested them for about a month in a fork (app ID hasn't changed, it was
just for testing): [cam-rod/org.signal.Signal_Beta](https://github.com/cam-rod/org.signal.Signal_Beta)

Also tagging the current maintainer @bermeitinger-b

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: org.signal.Signal.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+app-id: org.signal.Signal
```
Comment (hfiguiere) [inline org.signal.Signal.yaml:1]: this app already exist.

Comment (cam-rod) [reply org.signal.Signal.yaml:1]: Yes, but the beta branch has been unmaintained for a few years, so I would like to take responsibility for it.

Comment (hfiguiere) [reply org.signal.Signal.yaml:1]: then submit a PR on said repository

Comment (cam-rod) [reply org.signal.Signal.yaml:1]: Ok, will close this PR

**Pull request closed without merging**